src/Controllers/UserController.js

```diff
@@ -60,11 +60,18 @@ export class UserController extends AdaptableController {
       updateFields._email_verify_token_expires_at = {__op: 'Delete'};
     }
 
-    return this.config.database.update('_User', query, updateFields).then((document) => {
-      if (!document) {
-        throw undefined;
-      }
-      return Promise.resolve(document);
+    var self = this;
+    var checkIfAlreadyVerified = new RestQuery(self.config, Auth.master(self.config), '_User', {username: username, emailVerified: true});
```

Comment:
Oh yeah it doesn't really matter, the callback for UserController.verifyEmail doesn't actually take any arguments. But I'll change it to send back the document anyway.

In favor of keeping both in case `resend_verification_email` ever gets used by something else without first doing `verify_email` (though I doubt this will ever be the case so I guess we could remove it)